test/browser/env/default/default.properties

```diff
@@ -0,0 +1,25 @@
+# default.properties
+# properties set here will be available to the test execution as environment variables
+
+# sample_key = sample_value
+
+#The path to the gauge reports directory. Should be either relative to the project directory or an absolute path
+gauge_reports_dir = reports
```

Comment:
can this be set as .gauge-reports ?

Reply:
i think better to have it without dot so it's not hidden (no reason to hide)